### PR TITLE
added a blog: official account helper

### PR DESCRIPTION
**Pull request by sl1673495**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

想做一个公众号号主社群管理的行业解决方案， 之前按照 `starter` 的项目模板拉下来以后，发现依赖安装总是出各种问题，后来从零搭建并且手动添加依赖以后终于可以运行起来了。

把踩坑后总结的一个最小的可运行步骤记录下来，希望这篇博客可以帮助到和我一样运营公众号的号主们，感谢作者的贡献！

### CODE REVIEW

File: _posts/2020-07-01-a-guide-to-official-account-bot.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
+---
+
+ title: "使用wechaty助力公众号主维护社群"
+ date: 2020-07-01 12:30 +0800
+ author: sl1673495
```
Comment (lijiarui) [inline _posts/2020-07-01-a-guide-to-official-account-bot.md]: Please add a teaser for this blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Please change as I request! Thanks!

**Comment by sl1673495** [comment]
> Please change as I request! Thanks!

done, thanks!

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://github.com/wechaty/wechaty/wiki/Contributors), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Pull request merged**